index.js

```diff
@@ -0,0 +1,89 @@
+module.exports = {
+	root: true,
+	env: {
+		browser: true,
+		commonjs: true,
+		es6: true,
+		node: true,
+	},
+	globals: {
+		t: true,
+		n: true,
+		OC: true,
+		OCA: true,
+		Vue: true,
+		VueRouter: true
+	},
+	parserOptions: {
+		parser: 'babel-eslint',
+		ecmaVersion: 6
```

Comment:
Not needed? https://eslint.org/docs/user-guide/configuring#specifying-parser-options